tests/main/install-store/task.yaml

```diff
@@ -1,6 +1,6 @@
 summary: Checks for special cases of snap install from the store
 
-systems: [ubuntu-*]
+systems: [ubuntu-1*, ubuntu-2*]
```

Comment:
Hmm what did you mean here (especially with 2)

Reply:
I add a  comment:
# run on ubuntu-14,16,18,20+ but not on ubuntu-core, fedora etc